docs/src/admin-manual/configuration.md

```diff
@@ -0,0 +1,1262 @@
+AipCompressionLevel                          = 1
+AipCompressionAlgorithm                      = 6
+```
+
+For more detailed information on each of these configuration options, please
+consult the Archivematica documentation:
+
+* [Archivematica processing configuration fields]
+
+!!! important
+
+    Some of the options in the Archivematica documentation indicate that a given
+    setting can be left blank to prompt a user for a decision. **This will not
+    work with Enduro** - instead, the ingest workflow will eventually time out
+    while polling the perservation engine for updates, causing the workflow to
+    fail. For all but the last two configuration options listed above,
+    acceptable values are either `true` (i.e. "Yes" in the AM documentation) or
+    `false` (i.e. "No").
+
+    In most cases this should be clear, but there are a few configuration
+    options with more options supported in Archivematica than a3m. In such
+    cases, remember that currently in Enduro the preservation engine only
+    handles basic AIP creation - choose the options that align with that or else
+    those that involve skipping the processing step described.
+
+    For example, the **Normalize** processing configuration option in the
+    Archivematica docs includes a number of options, such as "Normalize for
+    preservation and access," "Normalize for access," "Normalize service files
+    for access," etc. A3m cannot at present generate
+    [DIPs](../user-manual/glossary.md#dissemination-information-package-dip) or
+    wait for manual normalization input - it can only normalize for
+    preservation. Consequently, `true` in this configuration file equates to
+    option 3 in the Archivematica docs (Normalize for preservation), while
+    `false` equates to option 7 (Do not normalize).
+
+### Archivematica configuration
+
+The next three configuration blocks must be set if you intend to use
+[Archivematica] as the [preservation engine] for ingest workflows.
+
+The settings in this first block define basic connection information.
+
+**Default values**:
+
+```toml
+[am]
+address = ""
+user = ""
+apiKey = ""
+processingConfig = "automated"
+capacity = 1
+pollInterval = "10s"
+transferDeadline = "1h"
+transferSourcePath = ""
+zipPIP = false
+```
+
+* `address`: Binds a3m to a specific address and port for communication. Because
+  [a3m] is the installation default preservation engine, this is blank until
+  configured.
+* `user`: Defines the username associated with the Archivematica instance. Blank
+  by default. For increased security, we recommend leaving this blank and
+  instead defining a persistent [environment variable] named `ENDURO_AM_USER`
+  with the chosen username.
+* `apiKey`: Defines the API key associated with the Archivematica instance.
+  Blank by default. For increased security, we recommend leaving this blank and
+  instead defining a persistent [environment variable] named `ENDURO_AM_APIKEY`
+  with the chosen key.
```

Comment:
In some cases it may be more secure to use an encrypted config file than the environment. We learnt this recently in a project where they were worried about a possible rogue admin user that could access the environment.

I think we should remove that recommendation here (from "For increased security...") and in the TOML config file. Not sure if we say something like that in other places.

Reply:
How can you encrypt a config file in a way that a rogue administrator couldn't read it, but the application can? :thinking:

Reply:
hey man, i just work here :joy: . i will remove the comment as requested by Radda, and add general recommendation at the top of the file about both encryption and environment variables.

Reply:
Lol, that was a question intended for @jraddaoui. ;)